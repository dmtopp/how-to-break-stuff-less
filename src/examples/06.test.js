import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Example from './06'

describe('(Component) Example 06', () => {
  it('should match a snapshot', () => {
    const component = shallow(<Example />)

    expect(toJson(component)).toMatchSnapshot()
  })
})
