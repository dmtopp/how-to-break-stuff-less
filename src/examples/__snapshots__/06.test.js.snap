// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Example 06 should match a snapshot 1`] = `
<main>
  <button>
    click me!
  </button>
</main>
`;
